Wabbajack.App.Wpf/Wabbajack.App.Wpf.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <OutputType>WinExe</OutputType>
+    <OutputType>Exe</OutputType>
```

Comment:
Any reason for this?

Reply:
I change that to get the console to popup. Will revert once PR is published.

If I can get the console without mucking with the proj file lemme know : )

Reply:
I assume you use Visual Studio? You can see this in the Output window at the bottom

Reply:
Figured I should be able to, just haven't been seeing it. I've probably hidden a window or something at some point.